charm/kafka/reactive/kafka.py

```diff
@@ -48,12 +48,19 @@ def upgrade_charm():
 def waiting_for_certificates():
     hookenv.status_set('waiting', 'waiting for easyrsa relation')
 
```

Comment:
Need another newline to pass `make lint`.. so we probably haven't run make lint yet...